remote/rapidjson/rapidjson.h

```diff
@@ -498,8 +498,12 @@ RAPIDJSON_NAMESPACE_END
 
 #ifndef RAPIDJSON_HAS_CXX11_RVALUE_REFS
 #if defined(__clang__)
-#define RAPIDJSON_HAS_CXX11_RVALUE_REFS __has_feature(cxx_rvalue_references) && \
+#if __has_feature(cxx_rvalue_references) && \

```

Comment:
rapidjson is a third party library that pxscene uses.  Please surround third party code changes with
/* MODIFIED CODE BEGIN */
/* MODIFIED CODE END */